crypto/async/arch/async_null.c

```diff
@@ -7,10 +7,12 @@
  * https://www.openssl.org/source/license.html
  */
 
-/* This must be the first #include file */
-#include "../async_locl.h"
```

Comment:
A little strange to see this include go away, but if the null implementation is not actually using anything, it sort-of makes sense.
On the other hand, it does provide the prototypes that are being added manually below -- does having it included actually cause problems?

Reply:
yes, it broke the cross-compile

Reply:
Because of windows.h?  Or the pragma?
Am I not supposed to explore alternative fixes as part of the review process?

Reply:
Sure you can.  :) In this case, start with the build-breakage report on the openssl-commits mailing list, https://mta.openssl.org/pipermail/openssl-commits/2017-August/016192.html